security/security.py

```diff
@@ -0,0 +1,184 @@
+from pydantic import BaseModel
+from typing import Union
+from fastapi.security import OAuth2PasswordBearer
+from passlib.context import CryptContext
+from datetime import datetime, timedelta
+from jose import JWTError, jwt
+from lib.utils import  load_config, load_json
+from fastapi import Depends, HTTPException, status
+
+
+# Load configuration from config.yaml file
+config = load_config('./config.yaml')
+
+# Set the secret key and algorithm for JWT token
+SECRET_KEY = config['secure']['SECRET_KEY']
+ALGORITHM = config['secure']['ALGORITHM']
+
+# Load dummy user database from JSON file
+db_dummy = load_json(r'./data/dummy_users_database.json')
+# Create a password context for hashing and verifying passwords
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+
+# Define the OAuth2 password bearer scheme for token authentication
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="token")
+
+# Define the Token model for representing the access token

```

Comment:
Same implementation you did here for the User but will all Data related structures, do not depend on dictionaries for complex function returns